packages/app/package.json

```diff
@@ -11,28 +11,20 @@
     "not dead"
   ],
   "scripts": {
-    "pre-commit": "run-s lint:staged test:static test:unit",
     "pre-push": "npm run ci:local",
-    "preflight": "npm install --legacy-peer-deps",
-    "develop": "gatsby develop --https -p 8000 --host local.viron.work",
+    "dev": "gatsby develop --https -p 8000 --host local.viron.work --open",
     "build": "gatsby build",
     "build:prefix": "gatsby build --prefix-paths",
     "serve": "gatsby serve -p 9000",
-    "serve:prefix": "gatsby serve -p 9000 --prefix-paths",
-    "launch": "run-s build serve",
-    "launch:prefix": "run-s build:prefix serve:prefix",
+    "launch": "npm run build && npm run serve",
     "clean": "gatsby clean",
-    "format": "run-p format:*",
-    "format:tjs": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
-    "lint": "npm run lint:tjs",
-    "lint:tjs": "eslint \"**/*.{js,jsx,ts,tsx}\"",
-    "lint:staged": "lint-staged",
-    "test": "run-s test:static test:unit test:e2e",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
+    "lint": "eslint \"**/*.{js,jsx,ts,tsx}\"",
     "test:e2e": "cypress run",
     "test:e2e:open": "cypress open",
     "test:e2e:record": "cypress run --record",
     "test:unit": "jest --passWithNoTests",
-    "test:static": "tsc",
+    "check-types": "tsc",
```

Comment:
There was no critical issue with the original script names. However, in projects generated with at least Turborepo v2.5.0, the same functionality is commonly named this way.

Following established external best practices, we will adopt the same naming convention.